lib/membrane_rtc_engine/endpoints/rtsp_endpoint.ex

```diff
@@ -38,19 +57,77 @@ if Enum.all?(
       state = %{
         rtc_engine: opts.rtc_engine,
         source_uri: opts.source_uri,
-        track: nil
+        stream_id: nil,
+        track_data: nil,
+        video: nil,
+        ssrc: nil
```

Comment:
I would put track struct here. Track has `ctx` field which we can use for storing internal data https://hexdocs.pm/membrane_rtc_engine/Membrane.RTC.Engine.Track.html#t:t/0